<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RootWindowW</class>
 <widget class="QWidget" name="RootWindowW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: hsl(210, 50%, 98%);
font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
  </property>
  <layout class="QHBoxLayout" name="HL_rootDivide" stretch="0,0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="w_mainMenu" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget
{
	background-color: hsl(210, 60%, 95%);
}
QPushButton
{
	color: hsl(0, 0%, 10%);
	font: 10pt &quot;微软雅黑&quot;;
	border-style: solid;
	border-width: 1px, 0px, 0px, 0px;
	border-radius: 3px;
	border-color: hsl(210, 5%, 90%);
	border-right-color: hsl(210, 5%, 100%);
	border-left-color: hsl(210, 5%, 100%);
	background-color: hsl(210, 5%, 100%);
}
QPushButton:hover
{
	color: hsl(0, 0%, 5%);
	background-color: hsl(210, 10%, 98%);
}
QPushButton:pressed
{
	color: hsl(0, 0%, 0%);
	background-color: hsl(210, 15%, 95%);
}
QPushButton:checked
{
	color: #ffffff;
	background-color: hsl(210, 100%, 40%);
}
QPushButton:checked:hover
{
	color: #ffffff;
	background-color: hsl(210, 80%, 45%);
}
QPushButton:checked:pressed
{
	color: #ffffff;
	background-color: hsl(210, 60%, 50%);
}
QPushButton:disabled
{
	color: hsl(210, 0%, 60%);
	border-width: 0px;
	/*border-color: hsl(210, 0%, 85%);*/
	background-color: hsl(210, 0%, 95%);
}</string>
     </property>
     <widget class="QPushButton" name="pb_tabbar_onlineSearcher">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>搜索</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
      <property name="pageID" stdset="0">
       <string>OnlineSearcher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_tabbar_localCollector">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>130</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>选择</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
      <property name="pageID" stdset="0">
       <string>LocalCollector</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_tabbar_itemCart">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>190</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>列表</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
      <property name="pageID" stdset="0">
       <string>ItemCart</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_tabbar_workQueue">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>250</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>工作</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
      <property name="pageID" stdset="0">
       <string>WorkQueue</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_tabbar_options">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>440</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>设置</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
      <property name="pageID" stdset="0">
       <string>options</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_progTitle">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>5</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 12pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <property name="text">
       <string>MUG</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lb_progStatus">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
	color: #ffffff;
	background-color: hsl(0, 0%, 40%);
}
QLabel[status=&quot;ready&quot;]
{
	background-color: hsl(210, 100%, 40%);
}
QLabel[status=&quot;working&quot;]
{
	background-color: hsl(200, 100%, 40%);
}
QLabel[status=&quot;warning&quot;]
{
	background-color: hsl(50, 100%, 40%);
}
QLabel[status=&quot;fatal&quot;]
{
	background-color: hsl(0, 100%, 40%);
}</string>
      </property>
      <property name="text">
       <string>就绪</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="status" stdset="0">
       <string>ready</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_progVersion">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>25</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Alp.3</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="w_funcWorkspace" native="true">
     <property name="styleSheet">
      <string notr="true">/*通用普通按钮*/
/*Normal Push Button*/

QPushButton
{
	color: hsl(0, 0%, 10%);
	border-style: solid;
	border-width: 1px;
	border-radius: 3px;
	border-color: hsl(210, 5%, 90%);
	background-color: hsl(210, 5%, 100%);
}
QPushButton:hover
{
	color: hsl(0, 0%, 5%);
	border-color: hsl(210, 10%, 88%);
	background-color: hsl(210, 10%, 98%);
}
QPushButton:pressed
{
	color: hsl(0, 0%, 0%);
	border-color: hsl(210, 15%, 85%);
	background-color: hsl(210, 15%, 95%);
}
QPushButton:disabled
{
	color: hsl(210, 0%, 60%);
	border-width: 0px;
	/*border-color: hsl(210, 0%, 85%);*/
	background-color: hsl(210, 0%, 95%);
}

/*通用高亮按钮*/
/*Highlighted Push Button*/

QPushButton[isHighlighted=true]
{
	color: #ffffff;
	border-style: solid;
	border-width: 1px;
	border-radius: 3px;
	border-color: hsl(210, 100%, 30%);
	background-color: hsl(210, 100%, 40%);
}
QPushButton[isHighlighted=true]:hover
{
	border-color: hsl(210, 80%, 35%);
	background-color: hsl(210, 80%, 45%);
}
QPushButton[isHighlighted=true]:pressed
{
	border-color: hsl(210, 60%, 40%);
	background-color: hsl(210, 60%, 50%);
}
QPushButton[isHighlighted=true]:disabled
{
	color: hsl(210, 0%, 100%);
	border-width: 0px;
	/*border-color: hsl(210, 0%, 60%);*/
	background-color: hsl(210, 0%, 70%);
}

/*通用行编辑框*/
/*Line Edit*/

QLineEdit
{
	color: hsl(0, 0%, 10%);
	selection-color: hsl(0, 0%, 100%);
	border-style: solid;
	border-width: 1px;
	border-radius: 3px;
	border-color: hsl(210, 5%, 90%);
	background-color: hsl(210, 5%, 100%);
	selection-background-color: hsl(210, 100%, 40%);
	padding: 0px 0px 0px 5px;
}
QLineEdit:hover
{
	color: hsl(0, 0%, 5%);
	border-color: hsl(210, 10%, 88%);
	background-color: hsl(210, 10%, 98%);
}
QLineEdit:focus
{
	border-bottom-color: hsl(210, 100%, 40%);
}
QLineEdit:disabled
{
	color: hsl(210, 0%, 60%);
	border-color: hsl(210, 0%, 85%);
	background-color: hsl(210, 0%, 95%);
}
QLineEdit:read-only
{
	border-color: hsl(210, 0%, 95%);
	background-color: hsl(210, 0%, 98%);
}

/*组合框及其附件*/
/*Combo box*/

QComboBox
{
	color: hsl(0, 0%, 10%);
	selection-color: hsl(0, 0%, 100%);
	border-style: solid;
	border-width: 1px;
	border-radius: 3px;
	border-color: hsl(210, 5%, 90%);
	background-color: hsl(210, 5%, 100%);
	selection-background-color: hsl(210, 100%, 40%);
	padding: 0px 0px 0px 5px;
}
QComboBox:hover
{
	color: hsl(0, 0%, 5%);
	border-color: hsl(210, 10%, 88%);
	background-color: hsl(210, 10%, 98%);
}
QComboBox:on
{
	border-color: hsl(210, 100%, 40%);
}
QComboBox:disabled
{
	color: hsl(210, 0%, 60%);
	border-color: hsl(210, 0%, 85%);
	background-color: hsl(210, 0%, 95%);
}
QComboBox::drop-down
{
	width: 0px;
	border: none;
}
QComboBox::down-arrow
{
	padding: 0px 32px 0px 0px;
	image: url(:/Resource/icons/drop-arrow-down);
}
QComboBox::down-arrow:on
{
	image: url(:/Resource/icons/drop-arrow-up);
}
QComboBox QAbstractItemView
{
	color: hsl(0, 0%, 10%);
	border-style: solid;
	border-width: 1px;
	border-radius: 3px;
	border-color: hsl(210, 100%, 40%);
	background-color: hsl(210, 5%, 100%);
	outline: 0px;
}
QComboBox QAbstractItemView::item
{
	color: hsl(0, 0%, 10%);
	border-style: none;
	background-color: hsl(210, 5%, 100%);
}
QComboBox QAbstractItemView::item:hover
{
	color: hsl(0, 0%, 5%);
	background-color: hsl(210, 10%, 98%);
}
QComboBox QAbstractItemView::item:selected
{
	color: #ffffff;
	background-color: hsl(210, 100%, 40%);
}

/*通用页面滑动条*/
/*Scroll Bar*/

QScrollBar
{
    background-color: hsl(210, 5%, 100%);
    padding: 12px 2px 12px 2px;
	width: 12px;
}
QScrollBar::handle
{
    background-color: hsl(210, 5%, 80%);
    border-radius: 2px; 
}
QScrollBar::handle:hover
{
    border-color: hsl(210, 10%, 78%);
}
QScrollBar::add-line:vertical
{
	height:12px;
	width:12px;
    border-image:url(:/Resource/icons/drop-arrow-down);
    subcontrol-position:bottom;
}
QScrollBar::sub-line:vertical
{
    height:12px;
	width:12px;
    image:url(:/Resource/icons/drop-arrow-up);
    subcontrol-position:top;
}

/*通用菜单内容*/
/*Menu &amp; Menu Content*/

QMenu
{
	color: hsl(0, 0%, 10%);
	border-style: solid;
	border-width: 1px;
	border-radius: 3px;
	border-color: hsl(210, 5%, 90%);
	background-color: hsl(210, 5%, 100%);
	padding: 5px 5px 5px 5px;
}
QMenu::item
{
	color: hsl(0, 0%, 10%);
	border-style: none;
	border-radius: 3px;
	background-color: hsl(210, 5%, 100%);
	padding: 4px 4px 4px 4px;
}
QMenu::item:selected
{
	color: hsl(0, 0%, 5%);
	background-color: hsl(210, 10%, 95%);
}
/* QMenu::item:selected
{
	color: #ffffff;
	background-color: hsl(210, 100%, 40%);
} */
QMenu::item:disabled
{
	color: hsl(210, 0%, 60%);
}
QMenu::separator
{
	height: 1px;
	background-color: hsl(210, 5%, 80%);
}
</string>
     </property>
     <layout class="QGridLayout" name="GL_workspaceSuit">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/MUG.qrc"/>
 </resources>
 <connections/>
</ui>
